test/integration/generated/lro/src/index.ts

```diff
@@ -7,6 +7,5 @@
  */
 
 export * from "./models";
-export { LROClient } from "./lROClient";
-export { LROClientContext } from "./lROClientContext";
+export { LROClient } from "./LROClient";
```

Comment:
Does the file name change look ok ? This should also because of Style.camel("LRO") = "LRO"

Reply:
The new name feels better!